rskj-core/src/test/java/co/rsk/peg/utils/Caller.java

```diff
@@ -0,0 +1,24 @@
+package co.rsk.peg.utils;
+
+import co.rsk.core.RskAddress;
+
+/**
+ * Created to makes easier the way to get RskAddress of 3 authorized and 1 unauthorized Callers. It
+ * contains hex encoded string associated to each caller and return the RskAddress of each one.
+ */
+public enum Caller {
```

Comment:
I wouldn't create this, at least not in a separate file. If it makes sense to have this enum then simply it put it inside the test class, it's the only place it's used

Reply:
But, we could use it in the UTs as well. We can think about having an isolated library project in the future for tests and create a `.jar` to be used as a kind of test `library/framework`, that could be used in `cross-projects (RSKj/Powpeg-Node)` and `cross-tests (UTs and ITs)`. In the meantime,  we might put all these objects together in a `utils` package. Creating it inside the class breaks the Single Responsibility Principle.

Reply:
I like @wilmerrootstock  idea very much about moving this out of the test for the benefit of reusability.

Reply:
Ok, in that case I'd move this to feeperkb package and rename it FeePerKbVoteCaller. What do you think?

I'm not a fan of generic `util` packages or classes

Reply:
Done!